source/chapters/appendix/mixxx_controls.rst

```diff
@@ -4376,6 +4016,414 @@ Controls
 
    :range: integer, read-only
 
+Deprecated controls
+~~~~~~~~~~~~~~~~~~~
+
+These controls have been deprecated and may be removed in a future version of Mixxx.
+In the meantime, skins and controller mappings that still use them will keep working, but using the suggested alternatives is strongly recommended.
+
+
+.. mixxx:control:: [Master],num_decks
+
+   The number of decks currently enabled.
+
+   :range: integer
+   :feedback: None
+
+   .. versionadded:: 1.9.0
+   .. deprecated:: 2.4.0
+       Use :mixxx:coref:`[App],num_decks` instead.
+
+
+.. mixxx:control:: [Master],num_samplers
+
+   The number of samplers currently enabled.
+
+   :range: integer
+   :feedback: None
+
+   .. versionadded:: 1.9.0
+   .. deprecated:: 2.4.0
+       Use :mixxx:coref:`[App],num_samplers` instead.
+
+
+.. mixxx:control:: [Master],num_preview_decks
+
+   The number of preview decks currently enabled.
+
+   :range: integer
+   :feedback: None
+
+   .. versionadded:: 1.9.0
+   .. deprecated:: 2.4.0
+       Use :mixxx:coref:`[App],num_preview_decks` instead.
+
+
+.. mixxx:control:: [Master],num_microphones
+
+   The number of microphone inputs that can be configured.
+
+   :range: integer
+   :feedback: None
+
+   .. versionadded:: 2.2.4
+   .. deprecated:: 2.4.0
+       Use :mixxx:coref:`[App],num_microphones` instead.
+
+
+.. mixxx:control:: [Master],num_auxiliaries
+
+   The number of auxiliary inputs that can be configured.
+
+   :range: integer
+   :feedback: None
+
+   .. versionadded:: 2.2.4
+   .. deprecated:: 2.4.0
+       Use :mixxx:coref:`[App],num_auxiliaries` instead.
+
+
+.. mixxx:control:: [Master],samplerate
+
+   The current output sample rate (default: 44100 Hz).
+
+   :range: absolute value (in Hz)
+   :feedback: None
+
+    .. deprecated:: 2.4.0
+       Use :mixxx:coref:`[App],samplerate` instead.
+
+
+.. mixxx:control:: [Master],headVolume
+
+    Adjust headphone volume.
+
+    :range: 0.0..1.0..5.0
+    :feedback: Headphone Gain knob
+
+    .. deprecated:: 2.0.0
+       Use :mixxx:coref:`[Master],headGain` instead.
+
+
+.. mixxx:control:: [Master],volume
+
+    Adjust main volume.
+
+    :range: 0.0..1.0..5.0
+    :feedback: Main Gain knob
+
+    .. deprecated:: 2.0.0
+       Use :mixxx:coref:`[Master],gain` instead.
```

Comment:
I guess [master] is not correct here.

Reply:
It is, we did not rename that yet. I just moved it.